ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -407,6 +459,13 @@ else
   echo "  Replicas: ${REPLICAS}"
 fi
 
+echo "  Enable Shared VPC: ${ENABLE_SHARED_VPC}"
+if [[ ${ENABLE_SHARED_VPC} == "yes" ]]; then
+  echo "    SAHRED_VPC_HOSTED_ZONE_ID: ${SAHRED_VPC_HOSTED_ZONE_ID}"
+  echo "    SAHRED_VPC_ROLE_ARN: ${SAHRED_VPC_ROLE_ARN}"
+  echo "    SAHRED_VPC_BASE_DOMAIN: ${SAHRED_VPC_BASE_DOMAIN}"
```

Comment:
Let us also record the information in the config file which may be used by the rosacli day2 validation. etc.
`record_cluster "audit_log_arn" $iam_role_arn`

Reply:
Updated